src/test/java/org/opentripplanner/middleware/TestUtils.java

```diff
@@ -115,13 +129,24 @@ public static boolean getBooleanEnvVar(String var) {
             if (otpUserFromDB != null && otpUserFromDB.applicationId != null) {
                 return headers;
             }
+            scope = OtpUser.SCOPE;
         }
 
         // Otherwise, get a valid oauth token for the user
-        headers.put("Authorization", "Bearer " + Auth0Users.getAuth0AccessToken(requestingUser.email, TEMP_AUTH0_USER_PASSWORD));
+        headers.put("Authorization", "Bearer " + getMockAuth0AccessToken(requestingUser.email, scope));
         return headers;
     }
 
+    private static String getMockAuth0AccessToken(String username, String scope) {
+        HttpResponse<String> response = Auth0Users.getApiUserAuth0TokenResponse(username, TestUtils.TEMP_AUTH0_USER_PASSWORD, scope);
```

Comment:
Hmm, if we're using this method to get a token with different scope values, maybe the method name `Auth0Users#getApiUserAuth0TokenResponse` doesn't quite work. Maybe a better one would be `Auth0Users#getAuth0TokenWithScope`?

Reply:
Updated.